shared/metrics/src/main/java/io/pravega/shared/metrics/MetricsConfig.java

```diff
@@ -22,6 +22,7 @@
     //region Config Names
     public final static Property<Boolean> ENABLE_STATISTICS = Property.named("enableStatistics", true);
     public final static Property<Long> DYNAMIC_CACHE_SIZE = Property.named("dynamicCacheSize", 10000000L);
+    public final static Property<Integer> DYNAMIC_CACHE_EVICTION_DURATION = Property.named("dynamicCacheEvictionDuration", 30);
```

Comment:
add the unit in the name so that its obvious what this value of 30 implies

Reply:
done.